### PR TITLE
Bug/vgnwb 223

### PR DESCRIPTION
**Pull request by tjquinn1**

### CODE REVIEW

**Review by prnvshrn — APPROVED** [review]
I have verified the changes on my local instance. The filters are retained when going to the next page in the 'Concepts' page.

File: concepts/views.py [hunk 1]
```diff
@@ -103,6 +104,18 @@ def concepts(request):
 
     paginator = Paginator(qs, 20)
     page = request.GET.get('page')
+
+    data = filtered.form.cleaned_data
+    params_data = {}
+    for key, value in data.items():
+        print key
```
Comment (hashknot) [inline concepts/views.py]: Is this 'print' statement needed here?

File: annotations/static/annotations/js/annotators/text.js [hunk 2]
```diff
@@ -100,8 +101,9 @@ var ConceptSearch = {
                 self.error = true;
                 self.searching = false;
             });
-        }
-    },
+        },
+        },
```
Comment (hashknot) [inline annotations/static/annotations/js/annotators/text.js]: You may want to indent this 4 spaces to the left.

File: concepts/views.py [hunk 3]
```diff
@@ -103,6 +104,18 @@ def concepts(request):
...
+        if key in ('typed'):
+           if value is not None and hasattr(value, 'id'):
+               params_data[key] = value.id
+        else:
+            if value is not None:
```
Comment (jdamerow) [inline concepts/views.py]: those two lines can be merged into an elif, can't it?

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: annotations/views/rest_views.py [hunk 4]
```diff
@@ -483,18 +484,24 @@ def search(self, request, **kwargs):
         if not q:
             return Response({'results': []})
         pos = request.GET.get('pos', None)
-
         concepts = goat.Concept.search(q=q, pos=pos, limit=50)
+
+
+
+
+
```
Comment (jdamerow) [inline annotations/views/rest_views.py]: remove excessive empty lines

File: annotations/views/repository_views.py [hunk 5]
```diff
@@ -368,6 +368,7 @@ def repository_text_content(request, repository_id, text_id, content_id):
 
 @login_required
 def repository_text_add_to_project(request, repository_id, text_id, project_id):
+    print "yes"
```
Comment (jdamerow) [inline annotations/views/repository_views.py]: remove

File: annotations/static/annotations/js/annotators/text.js (hunk 2 above)
Comment (jdamerow) [reply annotations/static/annotations/js/annotators/text.js]: yes

File: annotations/templates/annotations/project_details.html [hunk 6]
```diff
@@ -54,6 +54,7 @@
             </th>
         </thead>
         {% for text in texts %}
+        <h6>{{ text }}</h6>
```
Comment (jdamerow) [inline annotations/templates/annotations/project_details.html]: is this related to the ticket?

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Review by jdamerow — APPROVED** [review]
